.opspec/release/to-ghcr/op.yml

```diff
@@ -0,0 +1,95 @@
+name: to-docker
+description: releases opctl to [docker hub](https://hub.docker.com/r/opctl/opctl/)
+inputs:
+  github:
+    object:
+      constraints:
+        properties:
+          accessToken:
+            minLength: 1
+            type: string
+            writeOnly: true
+          username:
+            minLength: 1
+            type: string
+        required: [accessToken, username]
+      description: configuration required to interact w/ github
+  version:
+    string:
+      constraints: { format: semver }
+      description: version being published
+  dockerSocket:
+    socket:
+      description: docker socket
+    default: /var/run/docker.sock
+outputs:
+  alreadyPublished:
+    number:
+      description: version being published
+      constraints:
+        minimum: 0
+        maximum: 1
+run:
+  serial:
+    - op:
+        ref: github.com/opspec-pkgs/base64.encode#1.1.0
+        inputs:
+          rawValue: $(github.username):$(github.accessToken)
+        outputs:
+          encodedValue: $(b64GithubAuth)
+    - container:
```

Comment:
don't you want to do this inside release so you can skip both build, release to github and release to ghcr?

Reply:
Yep, good call. Moved it up.